packages/react-badge/src/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react';
+import { Meta } from '@storybook/react';
+import { Badge } from '../../Badge';
+import { AcceptIcon } from '../../tmp-icons.stories';
+import descriptionMd from '../../BadgeDescription.md';
+import bestPracticesMd from '../../BadgeBestPractices.md';
+
+export const Default = () => <Badge />;
+
+export const Sizes = () => {
```

Comment:
you'll need to create each story in its own `.stories.tsx` file and reexport from the main `Badge.stories.tsx` since that's how the codesandbox integration will work.

Also when you split up the stories, you'll need to use absolute imports too. You can use #19605 for reference

Reply:
Done